x/mint/keeper/genesis.go

```diff
@@ -21,15 +34,27 @@ func (keeper Keeper) InitGenesis(ctx context.Context, ak types.AccountKeeper, da
 
 // ExportGenesis returns a GenesisState for a given context and keeper.
 func (keeper Keeper) ExportGenesis(ctx context.Context) *types.GenesisState {
-	minter, err := keeper.Minter.Get(ctx)
+	params, err := keeper.Params.Get(ctx)
 	if err != nil {
 		panic(err)
 	}
 
-	params, err := keeper.Params.Get(ctx)
+	previousRewardEmissionPerUnitStakedToken, err := keeper.PreviousRewardEmissionPerUnitStakedToken.Get(ctx)
+	if err != nil {
+		panic(err)
+	}
```

Comment:
🤯

I know you are following the pattern but the pattern blows my mind.

Reply:
I agree, I don't like this about golang